docs/enterprise/console-settings.yaml

```diff
@@ -18,8 +18,20 @@ settings:
           ![The Sessions page in Pomerium Enterprise](../img/sessions-fullpage.png)
       - name: "Events"
         doc: |
+          The events page displays the log output of Envoy as it process changes from Pomerium and applies updates to the underlying services.
+
           ![The Events page in Pomerium Enterprise](../img/events-fullpage.png)
+
+          The most common updates are to Pomerium Proxy services, which are updated every time a Route or Policy is created or updated.
+
+          The value under **Resource ID** will usually match the resource ID of a [Policy][policy-reference], visible in the Policy under **Change History** or in the URL. A value of "Component reloaded" refers to when services are reloaded, usually due to a system update.
       - name: "Deployments"
+        doc: |
+          From the **Deployment History** page administrators can review changes made to their Pomerium configuration.
+
+          The default view shows all changes made through the Pomerium Enterprise Console. Use the **COMPARE** button next to an entry to filter to only changes that affected that resource. Select two versions of that resource, then **DIFF** to see what changed:
+
+          ![A screenshot showing the diff of a change to a route, adding a policy](../img/deployment-diff.png)
```

Comment:
@wasaga This is what I came up with after our discussion today on Events and Deployments. Please review.

Reply:
Updated and resolved in Slack.